vueapp/src/components/Game.vue

```diff
@@ -83,4 +86,11 @@ function moveGameToCourt() {
   selectedCourt.value.game = game.value;
   gameStore.removeFromOnDeck(game.value);
 }
+
+function completeGame(court) {
+  court.game.players.forEach(element => {
+    playerStore.addPlayer(element);
```

Comment:
Try to be more explicit with the naming of things: `player` is more intuitive in this case than `element`
```suggestion
    playerStore.addPlayer(player);
```